packages/mantine/src/index.ts

```diff
@@ -13,7 +13,7 @@ export * from './components';
 export * from './theme';
 
 // explicitly overriding mantine components
-export {Header, Table} from './components';
+export {Header, Table, Modal} from './components';
```

Comment:
Do we really want to override the Modal component? I think we should let the mantine modal be available directly without additional padding and footer.

We could name your component `Prompt` instead

Reply:
Yeah you're right, I changed to prompt !